<launch>
	<arg name="rviz" default="true" />

	<rosparam command="load" file="$(find lvio_lite)/config/mid360.yaml" />

	<param name="feature_extract_enable" type="bool" value="0"/>
	<param name="point_filter_num_" type="int" value="3"/>
	<param name="max_iteration" type="int" value="3" />
	<param name="filter_size_surf" type="double" value="0.5" />
	<param name="filter_size_map" type="double" value="0.5" />
	<param name="cube_side_length" type="double" value="1000" />
	<param name="runtime_pos_log_enable" type="bool" value="1" />
	<param name="preprocess/blind2" type="double" value="0.04" />
	<param name="pcd_save/pcd_save_en" type="bool" value="0" />
	<param name="ivox_nearby_type" type="int" value="6" />
	<param name="location_mode" type="bool" value="true" />
  <node pkg="lvio_lite" type="run_location_online" name="laserMapping" output="screen" />

	<group if="$(arg rviz)">
		<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find lvio_lite)/launch/include/local_rviz.rviz" />
	</group>

</launch>
